src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -354,7 +354,6 @@ pub const NI_NUMERICHOST: ::c_int = 0x00000002;
 pub const NI_NAMEREQD: ::c_int = 0x00000004;
 pub const NI_NUMERICSERV: ::c_int = 0x00000008;
 pub const NI_DGRAM: ::c_int = 0x00000010;
-pub const NI_NUMERICSCOPE: ::c_int = 0x00000020;
```

Comment:
You don't need to move it to the `freebsd12` dir since it was newly added.

Reply:
Do I then need to put it behind a `freebsd12` feature gate in `freebsd/mod.rs`? Or is that not needed here?

Reply:
`cfg`s e.g. `freebsd12` are for libc CI only and newly added constants are placed in `freebsd/mod.rs` usually (see other constants). CI will skip them by `skip_const` regardless of where they are (the `freebsd` dir vs. the `freebsdN` dir) but I'd like to keep them in the same place.

Reply:
Also, it'd be great if you could squash commits.

Reply:
I think that should be all